### DISCUSSION TITLE
Mention the difference about variables assignment and binding

### DISCUSSION THREAD

**Discussion by egze** [PJUllrich/run-elixir#3] [General]
The `Variables assignment` is technically not what's happening in Elixir. `=` is a match operator and not an assignment operator.

I get that for new people from other language it would sound confusing, but it's also not right to teach them incorrect concepts that could for example introduce awkward situations during a job interview.

Do you accepts PRs? I can take a stab at it to reword the text.

P.S. Love the idea and the website! Going to read the rest of the tutorial soon.

**Comment by PJUllrich** [comment]
Yes, you’re completely right. I decided against mentioning it to not confuse anybody. But I should have added it to the pattern matching chapter. If you wanna do that, please feel free to, but otherwise I’ll add it next week. On 7. Sep 2024, at 11:36, Aleksandr Lossenko ***@***.***> wrote:﻿
The Variables assignment is technically not what's happening in Elixir. = is a match operator and not an assignment operator.
I get that for new people from other language it would sound confusing, but it's also not right to teach them incorrect concepts that could for example introduce awkward situations during a job interview.
Do you accepts PRs? I can take a stab at it to reword the text.
P.S. Love the idea and the website! Going to read the rest of the tutorial soon.

**Comment by PJUllrich** [comment]
I added a new section in the `Pattern Matching` chapter :)

https://runelixir.com/pattern-matching.html#assignment-is-matching
